datana.app.version=Datata-PLC-Gateway-V2020-04-08_Array_Errors_With_IDs